drivers/sqljs-driver/build.gradle

```diff
@@ -0,0 +1,79 @@
+apply plugin: 'org.jetbrains.kotlin.multiplatform'
+
+kotlin {
+
+  js {
+    useCommonJs()
```

Comment:
I'm pretty sure this prevents use in a module system and we should be using UMD (since Kotlin can't emit ES6 module files).

Reply:
Done